app/experimenter/experiments/api/v5/query.py

```diff
@@ -1,31 +1,41 @@
 import graphene
 
-from experimenter.experiments.api.v5.types import NimbusExperimentType
+from experimenter.experiments.api.v5.types import (
+    NimbusExperimentIdOrder,
+    NimbusExperimentType,
+)
+from experimenter.experiments.constants.nimbus import NimbusConstants
 from experimenter.experiments.models.nimbus import NimbusExperiment
 
 
 class Query(graphene.ObjectType):
-    experiments_by_status = graphene.Field(
-        graphene.List(NimbusExperimentType),
-        status=graphene.Argument(
-            NimbusExperimentType._meta.fields["status"]._type, required=False
-        ),
-    )
-    all_experiments = graphene.List(
+    experiments = graphene.List(
         NimbusExperimentType,
+        description="List Nimbus Experiments.",
+        offset=graphene.Int(),
+        limit=graphene.Int(),
+        order=NimbusExperimentIdOrder(),
+        status=graphene.Enum(
+            "NimbusExperimentOptionalStatus", NimbusConstants.Status.choices
+        )(),
     )
     experiment_by_slug = graphene.Field(
-        NimbusExperimentType, slug=graphene.String(required=True)
+        NimbusExperimentType,
+        description="Retrieve a Nimbus experiment by its slug.",
+        slug=graphene.String(required=True),
     )
 
-    def resolve_all_experiments(root, info):
-        return NimbusExperiment.objects.all()
+    def resolve_experiments(
+        root, info, offset=0, limit=20, order="ascending", status=None
+    ):
+        q = NimbusExperiment.objects
+        q = q.order_by("-id") if order == "descending" else q.order_by("id")
```

Comment:
We can just refer to these string constants by reference from here, right?

https://github.com/mozilla/experimenter/pull/3677/files#diff-21992c79713e9ebfac085a293b207bbdR44-R45

In general I like to avoid hard coding duplicate string constants.

Reply:
Caveat: Except in tests where hard coding expectations is generally fine, I think.

Reply:
Ok, I'm just gonna remove it since we'll prolly have to sort differently later anyways.